hack/cr.yaml

```diff
@@ -6,10 +6,8 @@ metadata:
 spec:
   managementState: "Managed"
   nodeSpec:
-    image: quay.io/openshift/origin-logging-elasticsearch5:latest
```

Comment:
why not update this to be es6?

Reply:
It's one less place to update and if it matches what the operator expects is there a need to define it?